modules/network-observability-enabling-ebpf-metrics.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+// * network_observability/network-observability-operator-monitoring.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-netobserv-dashboard-ebpf-metrics-enable_{context}"]
+= Enabling eBPF metrics
+To see eBPF metrics on the *NetObserv/Health* dashboard, you must first enable them. 
+
+.Procedure
+. In the web console, navigate to *Operators* → *Installed Operators*.
```

Comment:
see other comment about `->`

Reply:
I actually need to remove this entire Procedure bc my engineers are making this enabled by default: https://github.com/netobserv/network-observability-operator/pull/649